scout/server/blueprints/variants/templates/variants/fusion-variants.html

```diff
@@ -0,0 +1,189 @@
+      <div id="collapseFilters" class="card-body panel-collapse collapse {{ 'show' if expand_search is sameas true }}">
+        {{ fusion_filters(form, institute, case) }}
+      </div>
+      {{ filter_form_footer(form, result_size, total_variants, more_variants, page, variants|length, institute) }}
+    </div>
+
+    <div class="card mt-3">
+      <table id="variantsTable" class="table table-hover table-bordered" aria-label="Fusion variants table">
+        <thead class="table-light thead">
+          <tr>
+            <th style="width:3%"></th>
+            <th style="width:4%" title="Beta version. Currently testing">Rank (beta)</th>
+            <th style="width:6%" title="HGNC symbols">Fusion Genes</th>
+            <th style="width:4%" title="Number of callers detecting a given event">Callers</th>
+            <th style="width:4%" title="Beta version. Currently testing">Score (beta)</th>
+            <th style="width:6%" title="Number of paired-ends that support the event">Junction Reads</th>
+            <th style="width:6%" title="Number of split reads that support the event">Split Reads</th>
+            <th style="width:6%" title="Fusion fragments per million total RNA-seq fragments">FFPM</th>
+            <th style="width:8%" title="Frame status of the fusion">Frame Status</th>
+            <th style="width:8%" title="Transcript ID">Transcript ID</th>
+            <th style="width:4%" title="Exon Number">Exon Number</th>
+            <th style="width:4%" title="Breakpoints">Breakpoints</th>
+            <th style="width:4%" title="Orientation">Orientation</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for variant in variants %}
+            {{ variant_row(variant) }}
+          {% else %}
+            <tr>
+                <td colspan="11">No matching variants</td>
+            </tr>
+          {% endfor %}
+        </tbody>
+      </table>
+    </div>
+  </div>
+  {{ dismiss_variants_block(dismiss_variant_options, institute, case, show_dismiss_block) }}
+</form>
+<div class="container-fluid" style="padding-bottom: 100px">
+ {{ pagination_footer(more_variants, page) }}
+</div>
+{% endblock %}
+
+{% macro variant_row(variant) %}
+  {% if variant.dismiss_variant %}
+   <tr class="dismiss">
+  {% elif 'causatives' in case and variant._id in case.causatives %}
+   <tr class="causative">
+  {% else %}
+  <tr>
+  {% endif %}
+    <td><input type="checkbox" value="{{variant._id}}" name="dismiss" onclick="enableDismiss()"></td>
+    <td class="text-start">
+      {{ cell_rank(variant, institute, case, form, manual_rank_options) }}
+    </td>
+    <td>
+      <div class="align-items-center">
+        {% if variant.genes %}
+          {% for gene in variant.genes %}
+             <a data-bs-toggle="tooltip" data-bs-html="true" title="
+               <div>
+                    <strong>{{ gene.hgnc_symbol or gene.hgnc_id }}</strong>:  {{ gene.description }}
```

Comment:
Ah see, it should display symbol or if symbol is missing then the HGNC ID.

**What I would expect on scout stage: display HGNC ID**, since HGNC symbol is missing in the demo databaae
**What I would expect in a non-demo instance: display symbo**l, since genes are found in the database